packages/scss/src/components/_skeleton.scss

```diff
@@ -0,0 +1,135 @@
+@use "sass:math";
+
+// Mixins
+
+@mixin skeletonShapeSizing($spacing) {
+	height: _theme("spacings.#{$spacing}");
+	width: _theme("spacings.#{$spacing}");
+}
+
+@mixin skeletonTextSizing($size) {
+	background-size: 150% math.div(_theme("sizes.#{$size}.font-size"), 2);
+	height: _theme("sizes.#{$size}.line-height");
+	&::before {
+		height: math.div(_theme("sizes.#{$size}.font-size"), 2);
+		top: calc(50% - math.div(_theme("sizes.#{$size}.font-size"), 4));
+	}
+}
+
+// Skeleton component styles
+
+.skeleton {
+	.skeleton-item {
+		animation: skeletonFade _theme("commons.animations.durations.fast")
+			ease-in 1 forwards;
+	}
+}
+
+.skeleton.is-loading {
+	
+	// Adds a pulse effet to skeleton container
+	&.has-pulse {
+		animation: skeletonPulse 1.5s infinite ease-in-out;
+	}
+	
+	// Default skeleton item
+	.skeleton-item {
+		animation: skeletonBackground 1.5s infinite linear;
+		background: linear-gradient(
+			90deg,
+			_color("grey", 50),
+			_color("grey", 200),
+			_color("grey", 50)
+		);
+		border-radius: _theme("commons.border.radius");
+		background-size: 150% 100%;
+		background-repeat: repeat-x;
+		display: block;
+		height: 100%;
+		opacity: 0;
+		width: var(--skltn-width, 100%);
+
+		// Shapes
+		&.mod-square,
+		&.mod-circle {
+			height: _theme("spacings.standard");
+			width: _theme("spacings.standard");
+
+			// Size modifiers
+			@each $name, $spacing in _getMap("spacings") {
+				&.mod-#{$name} {
+					@include skeletonShapeSizing($name);
+				}
+			}
+		}
+
+		&.mod-circle {
```

Comment:
.mod-rounded ? .mod-rounded-big ?

Reply:
Sorry I wasn't very explicit.

I think we can have:
- by default a rounding of 4,
- a mod for rounding of 8,
- a mod for rounding to 50%,
- a mod for rounding to 9999px,
- and a mod to remove rounding.